usr/share/rear/output/PXE/default/820_copy_to_net.sh

```diff
@@ -13,10 +13,14 @@ case "$scheme" in
         return 0
         ;;
     (fish|ftp|ftps|hftp|http|https|sftp)
+        if [[ ! "$OUTPUT_LFTP_OPTIONS" =~ ';'$ ]] ; then
```

Comment:
This condition also matches when $OUTPUT_LFTP_OPTIONS is empty
so that by default OUTPUT_LFTP_OPTIONS becomes a single `;` 
that is added to the `lftp` command.